packages/rawlink/rawlink.1.2/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+maintainer: "Christiano F. Haesbaert <haesbaert@haesbaert.org>"
+authors: "Christiano F. Haesbaert <haesbaert@haesbaert.org>"
+license: "ISC"
+homepage: "https://github.com/haesbaert/rawlink"
+bug-reports: "https://github.com/haesbaert/rawlink/issues"
+dev-repo: "git+https://github.com/haesbaert/rawlink.git"
+doc: "https://haesbaert.github.io/rawlink/"
+build: [
+  [ "dune" "subst" ] {dev}
+  [ "dune" "build" "-p" name "-j" jobs ] ]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {build}
```

Comment:
```suggestion
  "dune"
```
The build flag should not be used with dune for forward compatibility reasons

Reply:
Thanks